backend/core/src/test/java/org/sonarsource/sonarlint/core/BindingClueProviderTests.java

```diff
@@ -189,14 +190,144 @@ void ignore_scanner_file_invalid_content() {
     assertThat(logTester.logs(LogOutput.Level.ERROR)).contains("Unable to parse content of file 'file://path/to/sonar-project.properties'");
   }
 
+  @Test
```

Comment:
I suggest (pun intended) to write medium tests as much as possible, as per our guidelines. Unit tests should be used when it's too hard to test at the medium test level. Here I think some of the cases could be converted, some others are already covered by a medium test, and maybe some might stay as unit tests